docs/migration_guide.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Migration guide
-description: Guide to migrate pipelines using nf-validation pre v2.0.0 to after v2.0.0
+description: Guide to migrate pipelines using nf-schema pre v2.0.0 to after v2.0.0
```

Comment:
Should this guide actually be about how to migrate from `nf-validation` to `nf-schema`, rather than from an non-existent pre v2 of `nf-schema`?

Reply:
Yes it is actually, it seems I forgot to update the description though. Nice spot!

Reply:
I've updated the migration guide in #3 because a lot of stuff changed there. (Including your suggestion now)